docs/source/upcoming_release_notes/221-enh_status_out.rst

```diff
@@ -0,0 +1,23 @@
+221 enh_status_out
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- Adds results summary page accessible from run-mode
```

Comment:
Adding "in the `atef config` GUI" for future disambiguation could be a good thing, even if it's understood now